### ISSUE TITLE
401 Error

### ISSUE THREAD

**Issue by ghostbitmeta** [Douglas6/rockodi#1]
I keep getting "Could not connect [401]" but I am sure my IP, username and password is correct as I am able to log in using my browser.

**Comment by Douglas6** [comment]
If using a port other than 80, make sure you specify it in the Rockodi Configuration page in the IP Address field, e.g. '192.168.1.12:8080'.

**Comment by ghostbitmeta** [comment]
Yes I did, since I was testing it through my browser with the port as well.

**Comment by Douglas6** [comment]
Are you using 'special' characters (basically anything besides A-Z, a-z, or 0-9) in the user name or password? It's possible I need to do some extra escaping.

**Comment by ghostbitmeta** [comment]
I don't think it is that, my username and pass is [a-z]

**Comment by Douglas6** [comment]
I'm sorry you're experiencing these problems. Rockodi uses standard XMLHttpRequest methods; there may not be much I can do. I'll try and reproduce the problem. Are you pairing the Pebble with an Android or iPhone? Try clearing the username / password, or even reinstalling Rockodi.

**Comment by ghostbitmeta** [comment]
I am using an iPhone. 

I set the password blank on Kodi and it works that way.

**Comment by cjpowelldiz** [comment]
Ditto to this issue - using Pebble Time and iphone with Kodi on Win 10 64bit 
I typically use a custom port username and password - I found I could get the pebble remote to work by removing password if using custom port - it would be good if you can work on addressing this as I believe people who are likely to use a pebble and your app are likely to have custom port settings :)

**Comment by madeddie** [comment]
I use Kodi with custom port and username/password, but Android phone and Kodi on linux. Might be good to do some crosstesting (Android with Windows and iOS with Linux).
